components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -349,6 +382,107 @@ func handleBackupCommands(cmd *cobra.Command, args []string, commandString strin
 	return nil
 }
 
+var continueOnFailedBackup = `
+Backup failed with the following error: %v
+
+Do you wish to continue the backup?
+`
+
+func pathDiscrepancyErr(osPath, habp, abp, efLoc, complLoc, ingestLoc, erchefLoc string) error {
+	return fmt.Errorf(`discrepancy between backup paths. All backup paths should point to the same location:
+		path_repo: %s
+		backup_mount: %s
+		path: %s
+		event-feed-service location: %s
+		compliance-service location: %s
+		ingest-service location: %s
+		erchef-service location: %s
+		Edit the above to be the same and try again`, osPath, habp, abp, efLoc, complLoc, ingestLoc, erchefLoc)
```

Comment:
change this message `Edit the above to be the same and try again`
something like
`Please check the paths listed above, and please provide correct path.`